src/gather.ts

```diff
@@ -141,25 +150,32 @@ export class GatherProvider implements IGatherProvider {
 
         // Call internal slice method
         const slice = this._executionSlicer.sliceLatestExecution(gatherCell.persistentId);
-
-        return slice.cellSlices
+        const PpaResult = slice.cellSlices
           .reduce(concat, "")
           .replace(/#%%/g, defaultCellMarker)
           .trim();
-      }
+
+        if (PpaResult.length === 0) {
+          vscode.window.showErrorMessage(localize.Common.gatherError()  + '\n' + localize.Common.PPAError());
```

Comment:
Is there any information as to why the cell failed?

Reply:
Sadly no, I'd need to get the code form the user and debug it with Rob